x/src/cargo.rs

```diff
@@ -86,19 +88,35 @@ impl Cargo {
 
     pub fn run(&mut self) -> Result<()> {
         self.inner.stdout(Stdio::inherit()).stderr(Stdio::inherit());
-        self.do_run().map(|_| ())
+        self.do_run(true).map(|_| ())
```

Comment:
No need for this

Reply:
some way to default the parameter to true?